cognite/neat/workflows/inheritance_based/graph2assets_relationships.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, name: str, client: CogniteClient):
         self.transformation_rules: TransformationRules = None
         self.graph_source_type = "memory"
 
```

Comment:
@alivinco should we deprecate inheritance_based workflows ? and remove this module?